lib/modules/shared/widgets/flushbar.widget.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:another_flushbar/flushbar.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:project/config/theme.config.dart';
+
+enum MessageType { Confirmation, Information, Warning, Error }
+
+extension ParseToString on MessageType {
+  String toShortString() {
+    return this.toString().split('.').last;
+  }
+}
+
+void showFlushBar(BuildContext context,
+    {String message = '',
+    MessageType messageType = MessageType.Information,
+    VoidCallback? yesCallback,
+    VoidCallback? noCallback}) {
+  final iconsMap = {
+    MessageType.Confirmation: Icon(
+      Icons.quiz,
+      size: ThemeConfig.of(context).buttonHeight,
+      color: ThemeConfig.of(context).primaryColor,
+    ),
+    MessageType.Information:
+        Icon(Icons.info, size: ThemeConfig.of(context).buttonHeight, color: ThemeConfig.of(context).primaryColor),
+    MessageType.Warning:
+        Icon(Icons.warning, size: ThemeConfig.of(context).buttonHeight, color: ThemeConfig.of(context).warningColor),
+    MessageType.Error:
+        Icon(Icons.error, size: ThemeConfig.of(context).buttonHeight, color: ThemeConfig.of(context).errorColor)
+  };
+
+  final buttonTextStyle = ThemeConfig.of(context)
+      .subtitle2
+      .copyWith(color: ThemeConfig.of(context).primaryColor, letterSpacing: 1.25, fontWeight: FontWeight.w600);
+
+  var flushBar = null;
+
+  final confirmationLayout = Row(children: [
+    TextButton(
+        onPressed: () => {yesCallback!(), flushBar!.dismiss(true)},
```

Comment:
In dart arata putin diferit functiile lambda. Practic ai 2 feluri diferite:
#### expression lambda:
```
onPressed: () => myCallback(),
```

#### block body lambda:
```
onPressed: () {         // Fara '=>'
  myCallback();
},
```

Practic aici nu ai scris un lambda care executa 2 statement-uri, ai scris un lambda care returneaza un set care contine rezultatele celor 2 statement-uri.